### PR TITLE
Dums 57 tests

### PR DESCRIPTION
**Pull request by shewood**
### Description
Added tests for Player component. 

List of proposed changes:
added tests for Player component and fixed act warning on card component. 
updated skip link for room page

### CODE REVIEW

File: components/Card.test.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { render } from '@testing-library/react'
+import { render, act } from '@testing-library/react'
```
Comment (nibivi77) [inline components/Card.test.js:1]: We can remove `act` as it's not being used :)

**Review by nibivi77 — APPROVED** [review]
Good! Thank you Sheila!

File: components/Card.test.js (hunk 1 above)
Comment (shewood) [reply components/Card.test.js:1]: Good thing you noticed. I had added act to the test to fix the warning but it was removed on merge.

Comment (DaceyTom2) [reply components/Card.test.js:1]: I see act being used on line 25

File: pages/room/[id].js [hunk 2]
```diff
@@ -50,7 +50,7 @@ export default function Room(props) {
 
   return (
     <div
-      id="roomContent"
+      id="homeContent"
```
Comment (DaceyTom2) [inline pages/room/[id].js:53]: Not sure the purpose of changing this id

**Review by DaceyTom2 — APPROVED** [review]
Looks good, There is an ID in called homeContent being changed that looks accidentally

**Pull request merged**